eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -77,7 +77,8 @@ jobs:
     steps:
     - template: ../../../common/pipelines/templates/steps/verify-links.yml
       parameters:
-        Directory: "sdk/${{ parameters.ServiceDirectory }}"
+        Directory: ''
+        Urls: (git diff origin/master HEAD --name-only -- *.md)
```

Comment:
This seems to be a strange value to be input into a parameter called _Urls_.

Reply:
Urls can also point to local file paths :)